packages/flutter/lib/src/services/system_channels.dart

```diff
@@ -54,4 +54,10 @@ class SystemChannels {
       const JSONMessageCodec(),
   );
 
+  /// A JSON [BasicMessageChannel] for accessibility events.
```

Comment:
can we use something more efficient than json for this?

Reply:
I see everything else uses JSON too, that's sad.

Reply:
That's where I got the idea that JSON must be the right thing here :)

Changed it to `StandardMessageCodec`.